.github/workflows/add_issue_labels_on_open.yml

```
@@ -0,0 +1,43 @@
+on:
+  issues:
+    types: [opened]
+jobs:
+  apply-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v4
```

actionlint (1.7.9)

the runner of "actions/github-script@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)